chore/CheckJavadoc.java

```diff
@@ -0,0 +1,285 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+//DEPS fr.inria.gforge.spoon:spoon-core:10.4.2
+//DEPS fr.inria.gforge.spoon:spoon-javadoc:10.4.2
```

Comment:
Could we use the newest snapshot here? Otherwise, fixing bugs in spoon javadoc is not directly reflected in the check.

Reply:
How exactly?